### DISCUSSION TITLE
done notifications go to inbox when you look at them

### DISCUSSION THREAD

**Discussion by nt4f04uNd** [community/community#3700] [Mobile] [answered]
steps to reproduce
1. have a notification
2. mark it as "done"
3. view it
4. go back and check, it's gone to inbox

**Comment by nt4f04uNd** [comment]
it's very similar how on web unsubscrubing from "workflow" notifications moves them to "done", instead of removing them (which is yet again looks like a bug)

**Comment by brianlovin** [comment, marked as answer]
Hm @nt4f04uNd I can't reproduce this. I had a notification in my inbox, marked it as done, and it was removed from my inbox. From there I tried:

- going to notification filters → done → viewed the thread
- going to my issues → found the issue that triggered the notification → viewed the thread

In both cases, my notifications inbox was still empty.

Are there more specific steps to your repro that I'm missing?

**Reply by nt4f04uNd** [reply]
here's a video. is this intended?

https://user-images.githubusercontent.com/39104740/113746818-69302a80-970f-11eb-8eeb-c617c0a5489a.mp4

**Reply by nt4f04uNd** [reply]
(i can't repro this on ios)

**Reply by brianlovin** [reply]
Passing this along to the team - thanks so much for the video! This will be very useful for debugging.

**Reply by RSI4Life-Hub** [reply]
Clear

**Reply by aconsuegra** [reply]
thanks for the steps to reproduce the bug @nt4f04uNd. We have been able to find the root cause, and a fix will be available in our next update.
